### PR TITLE
Better align ANSSI kickstarts to their hardening levels

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Add or remove configurations from the ANSSI kickstarts to match with their profile hardening levels.

#### Rationale:

- The Kickstart and the Profile should be working together towards the same hardening level.

### CODE REVIEW

File: rhel7/kickstart/ssg-rhel7-anssi_nt28_intermediary-ks.cfg [hunk 1]
```diff
@@ -78,18 +78,14 @@ firewall --enabled --ssh
...
-
 # Set the system time zone (required)
 timezone --utc America/New_York
 
 # Specify how the bootloader should be installed (required)
 # Plaintext password is: password
 # Refer to e.g. https://pykickstart.readthedocs.io/en/latest/commands.html#rootpw to see how to create
 # encrypted password form for different plaintext password
-bootloader --location=mbr --append="crashkernel=auto rhgb quiet" --password=$6$zCPaBARiNlBYUAS7$40phthWpqvaPVz3QUeIK6n5qoazJDJD5Nlc9OKy5SyYoX9Rt4jFaLjzqJCwpgR4RVAEFSADsqQot0WKs5qNto0
+bootloader --location=mbr --password=$6$zCPaBARiNlBYUAS7$40phthWpqvaPVz3QUeIK6n5qoazJDJD5Nlc9OKy5SyYoX9Rt4jFaLjzqJCwpgR4RVAEFSADsqQot0WKs5qNto0
```
Comment (vojtapolasek) [inline rhel7/kickstart/ssg-rhel7-anssi_nt28_intermediary-ks.cfg:88]: I can see that you are not adding any boot parameter associated with audit here. Why not here and in High and Enhanced profile yes?

Comment (yuumasato) [reply rhel7/kickstart/ssg-rhel7-anssi_nt28_intermediary-ks.cfg:88]: Logging activity with audit is part of R50, which is part of Enhanced Level.

File: rhel8/kickstart/ssg-rhel8-anssi_bp28_enhanced-ks.cfg [hunk 2]
```diff
@@ -6,9 +6,6 @@
 # https://pykickstart.readthedocs.io/en/latest/
 # http://usgcb.nist.gov/usgcb/content/configuration/workstation-ks.cfg
 
-# Install a fresh new system (optional)
```
Comment (vojtapolasek) [inline rhel8/kickstart/ssg-rhel8-anssi_bp28_enhanced-ks.cfg:9]: What is the reaspon of removing this part? It stays in rhel7 kickstarts.

Comment (yuumasato) [reply rhel8/kickstart/ssg-rhel8-anssi_bp28_enhanced-ks.cfg:9]: Sorry, I should have mentioned that.
The `install` command is deprecated in RHEL8.
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_an_advanced_rhel_installation/kickstart-commands-and-options-reference_installing-rhel-as-an-experienced-user#install_kickstart-commands-for-installation-program-configuration-and-flow-control

Comment (yuumasato) [reply rhel8/kickstart/ssg-rhel8-anssi_bp28_enhanced-ks.cfg:9]: Having it doesn't cause any harm, but just a warning message.

Comment (vojtapolasek) [reply rhel8/kickstart/ssg-rhel8-anssi_bp28_enhanced-ks.cfg:9]: Thank you for explanation.

**Comment by vojtapolasek** [comment]
merging, openshift checks are not related to kickstarts.

**Pull request merged**